package/src/components/ShopLogo/v1/ShopLogo.js

```diff
@@ -26,13 +30,13 @@ export default class ShopLogo extends Component {
   }
 
   render() {
-    const { className, shopLogoUrl, shopName } = this.props;
+    const { className, height, shopLogoUrl, shopName } = this.props;
```

Comment:
Remove `height` from here

Reply:
My bad, that's a leftover from when I first wanted to add it as an actual prop. Then I realized that a theme prop would be more appropriate.